charts/newrelic-infrastructure/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: "DEPRECATED: (In maintenance mode) Helm chart for deploying New Relic Infrastructure Agent v2"
 name: newrelic-infrastructure
-version: 2.10.5
-appVersion: 2.13.2
+version: 2.10.6
+appVersion: 2.13.3
```

Comment:
If we are bumping the chart also, change the `verboseLog` helper to set `trace` to the agent.